libpages/server.go

```diff
@@ -21,14 +21,18 @@ import (
 	lru "github.com/hashicorp/golang-lru"
 	"github.com/keybase/kbfs/libfs"
 	"github.com/keybase/kbfs/libkbfs"
+	"github.com/keybase/kbfs/libmime"
 	"github.com/keybase/kbfs/libpages/config"
-	_ "github.com/keybase/kbfs/libpages/mime" // side-effects: add mime types
 	"github.com/keybase/kbfs/tlf"
 	"go.uber.org/zap"
 	"golang.org/x/crypto/acme"
 	"golang.org/x/crypto/acme/autocert"
 )
 
+func init() {
+	libmime.Patch(nil)
```

Comment:
Is there a reason not to put this in `ListenAndServe()`?  The port number is already hardcoded in that function, so it's not like it can be called twice.